src/components/planet.rs

```diff
@@ -7,3 +7,10 @@ pub struct Planet {
     pub planet: PlanetSprite,
     pub size: Vec2,
 }
+
+#[cfg(test)]
```

Comment:
Adding tests to components for creating a new struct that we'll use in the "spawn" of